src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.Internal/CliHelpers/DotnetCliRunner.cs

```diff
@@ -66,7 +76,16 @@ public int ExecuteAndCaptureOutput(out string? stdOut, out string? stdErr)
 
         process.EnableRaisingEvents = true;
 
-        process.Start();
+        try
+        {
+            process.Start();
+        }
+        catch (Exception e)
+        {
+            stdOut = e.Message;
```

Comment:
Not really sure about writing it out to both... but definitely better than writing it out to neither! (same for the callback scenario)

Reply:
I only had to stdErr earlier but figured if someone is debugging, they might only check one of those. Maybe just stdOut then?

Reply:
I think we probably just need to be consistent throughout the repo. If we're going to write errors to stderr, then we should do so. If we're going to write them to stdout in red, then we should do that. I tend to lean towards the latter because little of the ecosystem seems to use stderr, but I'm not particularly stuck on one or the other.

Reply:
ok makes sense, lets keep it consistent with stdErr